### ISSUE TITLE
Ruby 3.2.0 is in stable release, i can't find the list for ruby 3.2.0 stable in asdf list-all ruby

### ISSUE THREAD

**Issue by ariandanim** [asdf-vm/asdf-ruby#329]
Dear team,

Please make ruby 3.2.0 available in list-all ruby asdf, i see only 3.2.0 preview 3 in those list

**Comment by choosen** [comment]
https://github.com/asdf-vm/asdf-ruby/pull/322

**Comment by jcouball** [comment]
Try running `asdf plugin update ruby`.

```shell
$ asdf version
v0.11.1
$ asdf plugin update ruby
Updating ruby to master
Already on 'master'
Your branch is up to date with 'origin/master'.
$ asdf list all ruby | grep '3.2.0'
3.2.0-dev
3.2.0-preview1
3.2.0-preview2
3.2.0-preview3
3.2.0-rc1
3.2.0
jruby-9.3.2.0
$
```

**Comment by KelseyDH** [comment]
It's weird that you have to run `asdf plugin update ruby`, I was expecting `asdf update` to load the list of new ruby versions but it didn't.

**Comment by KelseyDH** [comment]
FYI Ruby `3.2.1` is now in stable release also.  When will that get added to asdf ruby?

**Comment by choosen** [comment]
@KelseyDH  it seems to plugin responsibility to handle updates per plugin.
I would have control on update: should it be only asdf engine or plugins also.
So maybe  `asdf update --plugins` ?  or some config for it ? :P

**Comment by pboling** [comment]
It would be cool if, like homebrew, asdf ruby would auto update sporadically.  Until then we must manually update the plugin.

**Comment by saraid** [comment]
It's really not asdf-ruby's responsibility to make those decisions. It's just a plugin for asdf-vm. If you want these enhancements, maybe suggest them on https://github.com/asdf-vm/asdf/issues ?

**Comment by SeanSith** [comment]
Agreed. @ariandanim this functionality does belong there. Can you close this and create an issue over there?

**Comment by choosen** [comment]
`asdf plugin-update --all`  seems to be the command , or just the plugin ?
waiting for ruby 2.7.8 to be released in ruby-build

**Comment by saraid** [comment]
@choosen `asdf plugin-update --all` appears to be defined here:

https://github.com/asdf-vm/asdf/blob/8919f4009ea233c32298911b28ceb879e2dbc675/lib/functions/plugins.bash#L116-L123

**Comment by choosen** [comment]
so we have hook named : `post_asdf_plugin_update`   and we can add most probably function to update ruby-build

**Comment by vladalive** [comment]
`asdf plugin update --all`
